Indriaty_Tutut/js/index.js

```diff
@@ -0,0 +1,90 @@
+import { SlideDeck } from './slidedeck.js';
+
+const map = L.map('map', {scrollWheelZoom: false}).setView([0, 0], 0);
```

Comment:
**Suggestion:** When the map first loads, it takes a long time to zoom into your area of focus. It may be better to have a `setView` here that is not `[0, 0], 0`. The numbers here mean `[latitude, longitude], zoom level`. You can zoom into a smaller area at the beginning when the zoom level is larger. See below for an example; feel free to play with these three numbers to achieve the effect you like.

```suggestion
const map = L.map('map', {scrollWheelZoom: false}).setView([-6.17504, 106.64339], 10);
```